api/api/util.py

```diff
@@ -294,3 +294,26 @@ def raise_if_exc(obj):
         _create_problem(obj)
     else:
         return obj
+
+
+from aiohttp import web
+
+def json_response_error_code(data=None, status=200, dumps=None, **kwargs):
```

Comment:
Is this function finally used?

Reply:
That function was deprecated. I have removed it